website/project/views/node.py

```diff
@@ -480,6 +479,14 @@ def view_project(auth, node, **kwargs):
     ret.update({'addons_widget_data': addons_widget_data})
     return ret
 
+
+@process_token_or_pass
+@must_be_valid_project(retractions_valid=True)
+@must_be_contributor_or_public
+def token_action(auth, node, **kwargs):
+    return redirect(node.url)
```

Comment:
Discussed in person issue with users who have an old email whose links won't work..but if the backend went out 48 hours before the front end it wouldn't be an issue. 

Looking at settings, it appears that these four email types all have links that expire in 48 hours, so it might be ok:


# Hours before pending embargo/retraction/registration automatically becomes active
RETRACTION_PENDING_TIME = datetime.timedelta(days=2)
EMBARGO_PENDING_TIME = datetime.timedelta(days=2)
EMBARGO_TERMINATION_PENDING_TIME = datetime.timedelta(days=2)
REGISTRATION_APPROVAL_TIME = datetime.timedelta(days=2)